src/lib/utils.ts

```diff
@@ -257,10 +257,9 @@ export const calculateBalance = (funds: ListfundsResponse): string => {
     return total.add(formatMsat(our_amount_msat))
   }, Big('0'))
 
-  // const onChain = funds.outputs.reduce((total, { amount_msat }) => total.add(amount_msat), Big('0'))
+  const onChain = funds.outputs.reduce((total, { amount_msat }) => total.add(amount_msat), Big('0'))
```

Comment:
```suggestion
  const onChain = funds.outputs.reduce((total, { amount_msat }) => total.add(formatMsat(amount_msat)), Big('0'))
```

<img width="752" alt="Screen Shot 2023-07-07 at 4 51 18 PM" src="https://github.com/clams-tech/browser-app/assets/30157175/167dffe2-acce-4d2d-bac9-cc68500183a9">